pkg/resource/deploy/source_eval.go

```diff
@@ -2197,7 +2227,28 @@ func (rm *resmon) RegisterResource(ctx context.Context,
 			Options: options,
 		})
 		if err != nil {
```

Comment:
We never hooked continue on error code properly into this but maybe we should?

Reply:
When we implemented continue on error, we decided to treat provider construct failures like programming errors and not continue at that point.  Given this work maybe we should revisit that, though to me it still feels like most of the errors coming back from here are users programming errors, and thus we should not continue when they happen?

Reply:
Treated more like Check than Create/Update. That seems reasonable for now.